### PR TITLE
Ss/tagfilter

### PR DESCRIPTION
**Pull request by obct537**

### CODE REVIEW

File: castle/cms/tiles/querylisting.py [hunk 1]
```diff
@@ -92,6 +92,14 @@ class TaggedView(BaseTileView):
     tile_name = 'querylisting'
 
 
+class TagFilterView(BaseTileView):
+    name = 'tag-filter'
+    preview = '++plone++castle/images/previews/querylisting/tagged.png'
```
Comment (vangheem) [inline castle/cms/tiles/querylisting.py]: missing file?

File: castle/cms/tiles/querylisting.py [hunk 2]
```diff
@@ -268,6 +276,8 @@ def filter_pattern_config(self):
                 isinstance(config['query']['Subject'], basestring)):
             config['query']['Subject'] = [config['query']['Subject']]
 
+        config['display_type'] = self.data.get('display_type', None)
```
Comment (vangheem) [inline castle/cms/tiles/querylisting.py]: clever using this for pattern config

File: castle/cms/tiles/querylisting.py (hunk 1 above)
Comment (obct537) [reply castle/cms/tiles/querylisting.py]: crap, good catch.

Comment (obct537) [reply castle/cms/tiles/querylisting.py]: added a new preview image

**Pull request merged**